src/bindgen/cdecl.rs

```diff
@@ -286,7 +286,7 @@ pub fn write_func<F: Write>(
 }
 
 pub fn write_field<F: Write>(out: &mut SourceWriter<F>, t: &Type, ident: &str) {
-    CDecl::from_type(t).write(out, Some(ident), false);
+    CDecl::from_type(t).write(out, Some(ident), true);
```

Comment:
In other places we intentionally _don't_ do this for C++, so I think we should be consistent here.

Most of the callers if not all should have a `config` object. I think we should pass it through and always write the `void` when the language is "C", wdyt?

Reply:
That's a good idea, I'll apply these change later.